vendor/urls/vendor_admin.py

```diff
@@ -15,8 +15,14 @@
     path('offer/', admin_views.AdminOfferCreateView.as_view(), name="manager-offer-create"),
     path('orders/', admin_views.AdminInvoiceListView.as_view(), name="manager-order-list"),
     path('order/<uuid:uuid>/', admin_views.AdminInvoiceDetailView.as_view(), name="manager-order-detail"),
+    path('subscriptions/', admin_views.AdminSubscriptionListView.as_view(), name="manager-subscriptions"),
+    path('subscription/<uuid:uuid>', admin_views.AdminSubscriptionDetailView.as_view(), name="manager-subscription"),
+    path('profiles/', admin_views.AdminProfileListView.as_view(), name="manager-profiles"),
+    path('profile/<int:pk>', admin_views.AdminProfileDetailView.as_view(), name="manager-profile"),
```

Comment:
can we use UUIDs instead of PKs?  Or even usernames?

Reply:
Added uuid for address and CustomerProfile and corresponding migrations